native/packages/react-native-bpk-component-card/readme.md

```diff
@@ -62,11 +62,90 @@ export default class App extends Component {
 }
 ```
 
+### `withDivider` HOC
+
+```js
+import React, { Component } from 'react';
+import { View, StyleSheet, Text } from 'react-native';
+import BpkText from 'react-native-bpk-component-text';
+import * as TOKENS from 'bpk-tokens/tokens/ios/base.react.native.es6';
```

Comment:
We don't wanna encourage this type of import right?

Reply:
Indeed - fixed.